examples/physical_robot_remote/README.md

```diff
@@ -0,0 +1,12 @@
+In this example, you will control a physical modular robot by running its brain locally on your computer and streaming movements commands to the physical modular robot.
+
+You learn:
+- How to create a configuration describing how a physical modular robot should be controlled.
+- How to remote control a physical modular robot using this configuration.
+
+Requirements:
+- A physical modular robot. Not whether you are using V1 or V2 hardware; If you do not know, it is probably V2. TODO in the next version of Revolve2 a guide will be added on how to build a physical robot. For now, ask the CI Group lab.
```

Comment:
What does the "Not whether you are using" mean?

Reply:
'note' :)